### PR TITLE
Adding custom links to the dashboards

### PR DESCRIPTION
**Pull request by ivangonzalezacuna**
As discussed, I'm adding the option to insert custom links into the dashboards card. I have tested it and it works fine. In case no tag is added, it just returns an empty array, so no issues have been found.

I have also removed the check for the missing annotation, as I'm adding a Warning in out backstage instance

### CODE REVIEW

File: src/components/DashboardsCard/DashboardsCard.tsx [hunk 1]
```diff
@@ -93,9 +93,22 @@ const Dashboards = ({
...
+    grafanaUrls.forEach(g => {
+      const parts = g.split('::', 2);
+      grafanaDashboards.push({
+        id: Date.now() + Math.random(),
+        url: parts[1],
+        title: parts[0],
+        folderUrl: "",
+        folderTitle: ''
+      });
+    });
```
Comment (heyLu) [inline src/components/DashboardsCard/DashboardsCard.tsx]: This seems very specific to how we use the plugin, can you try to instead pass a hook to this component that can add additional dashboards based on the entity?  Then we could move this code (or similar code) to our Backstage installation and leave more flexibility to this plugin.

File: src/components/DashboardsCard/DashboardsCard.tsx [hunk 2]
```diff
@@ -117,10 +130,5 @@ export type DashboardCardOpts = {
 
 export const DashboardsCard = (opts?: DashboardCardOpts) => {
   const { entity } = useEntity();
-
-  return !isGrafanaAvailable(entity) ? (
-    <MissingAnnotationEmptyState annotation={GRAFANA_ANNOTATION_TAG_SELECTOR} />
-  ) : (
-    <Dashboards entity={entity} opts={opts} />
-  );
```
Comment (heyLu) [inline src/components/DashboardsCard/DashboardsCard.tsx]: I think we should keep this and just require people that want Grafana links to add the basic `grafana/tag-selector` annotation, like we do for our other integrations.

**Review by heyLu — CHANGES_REQUESTED** [review]

File: src/components/DashboardsCard/DashboardsCard.tsx (hunk 2 above)
Comment (ivangonzalezacuna) [reply src/components/DashboardsCard/DashboardsCard.tsx]: Okay, so I will keep it as it was before

File: src/components/DashboardsCard/DashboardsCard.tsx (hunk 1 above)
Comment (ivangonzalezacuna) [reply src/components/DashboardsCard/DashboardsCard.tsx]: Okay, I get the idea. I will try this solution then

**Comment by ivangonzalezacuna** [comment]
Applied the changes. I had to export the dashboard to be able to use the hook. It follows the same idea as for the `EntitySwitch.Case` (for example)

**Review by heyLu — APPROVED** [review]
Ok, this looks nicer now!  When we finally submit this upstream we might have to rework this further, but for now this is a nice simple solution. ✨

**Pull request merged**